build.sbt

```diff
@@ -24,39 +22,8 @@ inThisBuild(Seq(
   organization := "com.lucidchart",
   scalaVersion := "2.11.12",
   scmInfo := Some(ScmInfo(url("https://github.com/lucidsoftware/piezo"), "scm:git:git@github.com:lucidsoftware/piezo.git")),
-  version := sys.props.getOrElse("build.version", "0-SNAPSHOT")
+  version := sys.props.getOrElse("build.version", "0-SNAPSHOT"),
+  versionScheme := Some("early-semver"),
 ))
 
-val bintrayDescriptor = taskKey[File]("Descriptor for TravisCI release to Bintray")
-
-bintrayDescriptor in (ThisBuild, Debian) := {
-  def files(deb: File) = Json.obj(
-    "includePattern" -> baseDirectory.value.toPath.relativize(deb.toPath.normalize).toString,
-    "matrixParams" -> Json.obj(
-      "deb_architecture" -> "amd64,i386",
-      "deb_component" -> "main",
-      "deb_distribution" -> "piezo"
-    ),
-    "uploadPattern" -> s"pool/p/piezo-admin_${version.value}_all.deb"
-  )
-  val json = Json.obj(
-    "files" -> Json.arr(
-      files((packageBin in (admin, Debian)).value)
-    ),
-    "package" -> Json.obj(
-      "name" -> "piezo",
-      "repo" -> "apt",
-      "subject" -> "lucidsoftware"
-    ),
-    "publish" -> true,
-    "version" -> Json.obj(
-      "name" -> version.value,
-      "gpgSign" -> true,
-      "vcs_tag" -> version.value
-    )
-  )
-  val file = target.value / Debian.name / "bintray.json"
-  file.getParentFile.mkdirs()
-  IO.write(file, Json.prettyPrint(json))
-  file
-}
+publishTo := sonatypePublishToBundle.value
```

Comment:
do we still need this?

Reply:
might be better to some one reuse commonSettings

Reply:
I think so, but I'm not 100% sure.